services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusServiceITest.kt

```diff
@@ -2002,6 +2005,30 @@ class HakemusServiceITest(
             verifySequence {
                 alluClient.create(any())
                 alluClient.addAttachment(alluId, any())
+                alluClient.sendSystemComment(alluId, any())
+                alluClient.getApplicationInformation(alluId)
+            }
+        }
+
+        @Test
+        fun `sends a system comment to Allu when there is a paper decision receiver`() {
```

Comment:
Where does the name "system comment" come from? In Allu API description they are called "application comments". Okay, now I see it: in Allu UI comments are categorized and external system is one category: "Haitaton järjestelmä 4.10.2024 9.27 Ulkoinen järjestelmä". Btw, "Haitaton järjestelmä" seems a bit faulty as Finnish term - maybe "Haitaton-järjestelmä" could be better? Or is "Haitaton järjestelmä" already a "standard"?

Reply:
It's called a system comment because it's created by Haitaton itself. The same commenting API is envisioned to be used for two-way communication between the applicant and the Allu handlers.

Reply:
I changed the sender to "Haitaton-järjestelmä".